extensions/common/api/management-api-configuration/src/main/java/org/eclipse/edc/connector/api/management/configuration/ManagementApiConfigurationExtension.java

```diff
@@ -105,5 +110,8 @@ public void initialize(ServiceExtensionContext context) {
 
         context.registerService(ManagementApiConfiguration.class, new ManagementApiConfiguration(webServiceConfiguration));
         webService.registerResource(webServiceConfiguration.getContextAlias(), new AuthenticationRequestFilter(authenticationService));
+
+        var jsonLdMapper = typeManager.getMapper("json-ld");
```

Comment:
nit: there is a constant for this in `JsonLdExtension.java`, which - if moved out to maybe `CoreConstants.java` or something, could be re-used.

Reply:
Not a huge fan of global generic purpose constants, however for the moment I put it there as suggested